deployer/src/persistence/mod.rs

```diff
@@ -549,6 +566,36 @@ mod tests {
         );
     }
 
+    #[tokio::test(flavor = "multi_thread")]
+    async fn get_deployments() {
```

Comment:
Thanks for adding a test. What would happen if we pass an out-of-bounds `offset`?

Reply:
An out-of-bounds `offset` where? If someone makes an API request with a value that can't be parsed into a `u32`, the handler won't be called and the user will receive an error from Axum. I'm also pretty sure that the database will accept any `u32` as an `OFFSET` for a query, so that should be fine.

Reply:
Yeah, the concern was more toward the fact that the OFFSET can be bigger than the total number of rows. However, @Kazy assured me the result will still be an empty row list and error-free. I will keep this comment open still because we agreed to add in the tests a case for querying for an offset bigger than the rows count.